src/AddressRegistry.Api.Legacy/Address/AddressController.cs

```diff
@@ -141,7 +141,7 @@ public class AddressController : ApiController
         /// <param name="taal">Gewenste taal van de respons.</param>
         /// <param name="responseOptions"></param>
         /// <param name="cancellationToken"></param>
-        /// <response code="200">Als de opvraging van een lijst met adressen gelukt is.</response>
+        /// <response code="200">Als de adresmatch gelukt is. </response>
```

Comment:
Can you remove the whitespace after the `.` ?

Reply:
done